styles/forms.css

```diff
@@ -0,0 +1,7 @@
+.form-control,
+.form-select {
+  &:focus {
+    border-color: var(--stanford-black);
+    box-shadow: 0 0 0 2px var(--stanford-black);
```

Comment:
should these both be black?

Reply:
I don't see that here: https://github.com/sul-dlss/component-library/issues/22#issuecomment-3614369765

Reply:
ok. happy to pair on this if helpful